create_blog_post_using_azure_openai_and_publish_in_medium/main.bal

```diff
@@ -0,0 +1,45 @@
+import ballerina/io;
+import ballerinax/azure.openai.text;
+import ballerinax/medium;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string mediumToken = ?;
+
+const API_VERSION = "2023-03-15-preview";
+
+public function main(string title) returns error? {
+    final medium:Client medium = check new ({auth: {token: mediumToken}});
```

Comment:
```suggestion
    final medium:Client mediumClient = check new ({auth: {token: mediumToken}});
```

Reply:
When we did the previous samples, @sameerajayasoma suggested that we define the client variables without the `Client` suffix to better represent the service.

Reply:
@JayaniH Ack. +1 to keep it as is